ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/clientlibs/site/js/accordionview.js

```diff
@@ -135,8 +135,22 @@
             return this.element.querySelector(Accordion.selectors.qm);
         }
 
-        setFocus() {
+        setFocus(id) {
             this.setActive();
+            this.#collapseAllItems();
+            const item = this.#getItemById(id + '-item');
```

Comment:
won't we need the super.setFocus call , otherwise how will focus flow to parent ??

Reply:
this is the same for wizard and tabs also

Reply:
done